apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -134,7 +137,9 @@ get_current_block(Peer, BI) ->
 
 %% @doc Retreive a block by height or hash from a remote peer.
 get_block(Peer, ID, BI) ->
-	case get_full_block([Peer], ID, BI) of
+	get_block(Peer, ID, BI, ?WITH_TX_DATA).
+get_block(Peer, ID, BI, TXFormat) ->
```

Comment:
It looks like this one is also only used in `ar_http_iface_tests`, should we replace `get_full_block` with it? This would require to change all the places that call `get_full_block` then, but it might be worth it.

Reply:
I'm sure we can. I'll update